### DISCUSSION TITLE
Zugang zu config.ini wird verwehrt.

### DISCUSSION THREAD

**Discussion by stiech82** [fabian-lauer/dbus-shelly-3em-smartmeter#51] [Q&A]
Hallo.

Ich habe mit Erfolg dbus-opendtu installieren können und da kann ich auf die config.ini zugreifen.
Jetzt mit dem dbus-shelly-3em-smartmeter bin ich genau so vorgegangen wie in der Anleitung hier steht, aber wenn ich die config.ini öffnen möchte wird mir der Zugang verwehrt.

-sh: /data/dbus-shelly-3em-smartmeter/config.ini: Permission denied

Ich habe einen original Cerbo GX mit der Firmwareversion V3.00

**Comment by KapitainNemo** [comment]
@stiech82 , hast du die config.ini editieren können und wenn ja, wie?
